packages/extension/src/ui/components/IconBar.tsx

```diff
@@ -35,7 +36,7 @@ export const IconBar: FC<IconBarProps> = ({ back, close }) => (
     )}
     <hr />
     {close && (
-      <Link to={isString(close) ? close : routes.accountTokens()}>
+      <Link to={close !== undefined ? (close as any) : routes.accountTokens()}>
```

Comment:
```suggestion
      <Link to={isString(close) || close === -1 ? (close as any) : routes.accountTokens()}>
```
We shouldn't check for undefined here because we already checking it on line 38.